activities/postboard/client/styles.css

```
@@ -0,0 +1,327 @@
+  width: min(30rem, 100%);
+  margin: 1rem 0 1rem auto;
+}
+
+.postboard-sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)